### PR TITLE
Define our goals.

### PR DESCRIPTION
**Pull request by hober**
Splitting #3 up into one PR per principle, so we can quickly land the uncontroversial ones, and have focused discussion on the others.

### CODE REVIEW

**Comment by hober** [comment]
Reviews very welcome, @kewisch @jessieberlin @dotproto @cwilso @tantek @MaheshKumarJha!

**Review by tantek — CHANGES_REQUESTED** [review]
These goals are a good summary. One nit, since it looks like this is the first time the seemingly generic phrase "web extensions" is used in the document, it may benefit from a more precise description, noting that the rest of the doc will use "web extensions" as that shorthand.

E.g.

s/core of APIs for web extensions./core of APIs for web browser extensions (hereafter summarily referred to as "web extensions").

Feel free to wordsmith. I think by making it clear at the start that we're talking about web *browser* extensions, and not some generic model of extensions for all web technologies (like a couple of TLAs that shall not be named), it'll help reduce confusion when read by a broader audience.

File: charter.md [hunk 1]
```diff
@@ -38,6 +38,12 @@ The WebExtensions Community Group defines [WebExtensions](#web-extensions).
 
 ## Goals
 
+We will specify a model, permissions, and a common core of APIs for web
+extensions. By specifying the APIs, functionality, and permissions of
```
Comment (kewisch) [inline charter.md:42]:
```suggestion
We will specify a model, permissions, and a common core of APIs for WebExtensions. By specifying the APIs, functionality, and permissions of
```

**Comment by kewisch** [comment]
I like Tantek's suggestion and as mentioned in another PR would like us to use WebExtensions instead of web extensions.

**Review by kewisch — APPROVED** [review]

**Comment by hober** [comment]
@tantek @kewisch okay, I think I've fixed this in e7fe561. Please take another look.

**Review by tantek — APPROVED** [review]
LGTM

**Comment by kewisch** [comment]
LGTM!

**Pull request merged**